.github/workflows/release.yml

```diff
@@ -0,0 +1,107 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      publishPreRelease:
+        description: 'Publish a pre-release ?'
+        required: true
+        type: choice
+        options:
+          - 'true'
+          - 'false'
+        default: 'false'
```

Comment:
I would remove this. We don't support pre-releases here either, and the logic to do it is missing anyways.

Reply:
We did support prerelease in `YAML` in `Jenkins` earlier